makefiles/misc-cmds.mk

```
@@ -0,0 +1,51 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += misc-cmds
+MISC-CMDS_VERSION := 34
+DEB_MISC-CMDS_V   ?= $(MISC-CMDS_VERSION)
+
+misc-cmds-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://opensource.apple.com/tarballs/misc_cmds/misc_cmds-$(MISC-CMDS_VERSION).tar.gz
+	$(call EXTRACT_TAR,misc_cmds-$(MISC-CMDS_VERSION).tar.gz,misc_cmds-$(MISC-CMDS_VERSION),misc-cmds)
+	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin
```

Create directories from lines 22-23.
```suggestion
	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/{bin,share/{man/man1,misc}}
```